src/lang/en-US.ts

```diff
@@ -36,6 +37,10 @@ const enUSMessages: ResolvedIntlConfig['messages'] = {
     'data.email': `Email`,
     'data.display_name': `User Name`,
 
+    // Confirmations
+    'confirm.title': `Are you sure?`,
+    'confirm.loseData': `You have unsaved worked. If you continue you will lose your changes.`,
```

Comment:
The first sentence of the `confirm.loseData` message should be _You have unsaved work._ Currently, the string is using the wrong tense for the verb _work_.